docs/python/settings-reference.md

```diff
@@ -15,12 +15,12 @@ MetaSocialImage: TBD
 
 | Setting | Default | Description |
 | --- | --- | --- |
-| python.pythonPath | `"Python"` | Path to the python interpreter. |
-| python.venvPath | `""` | Path to folder with a list of Virtual Environments, such as ~/.pyenv, ~/Envs, ~/.virtualenvs. |
+| python.pythonPath | `"python"` | Path to the python interpreter. |
```

Comment:
Here's an example where it's not python3, just python.

Reply:
That's to support Windows and its lack of `python3` command.